tests/beam_design/test_beam_design.py

```diff
@@ -0,0 +1,24 @@
+
+from lebedigital.BeamDesign import BeamDesign
+from BeamDesign import *
```

Comment:
import * is not encouraged, see e.g.: [why import star in python is a bad idea](https://www.geeksforgeeks.org/why-import-star-in-python-is-a-bad-idea/)
better import BeamDesign and then call the functions as BeamDesign.functionName, or specify specifically which functions you need.

Reply:
i will change that